guides/micronaut-java-records/java/src/main/java/example/micronaut/BookForSale.java

```diff
@@ -0,0 +1,13 @@
+package example.micronaut;
+
+import io.micronaut.core.annotation.NonNull;
+import io.micronaut.core.annotation.Introspected;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import java.math.BigDecimal;
+
+@Introspected // <1>
+public record BookForSale(@NonNull @NotBlank String isbn,  // <2>
+                          @NonNull @NotBlank String title,  // <2>
+                          @NonNull @NotNull BigDecimal price) {  // <2> 
+                                                    }
```

Comment:
`price` callout is not rendered properly:

![image](https://user-images.githubusercontent.com/559192/142606671-9ff1b5de-4852-46eb-8b55-1a3a5155ddeb.png)